test/test_datapipe.py

```diff
@@ -35,6 +41,63 @@
 
 T_co = TypeVar('T_co', covariant=True)
 
+PORT = 8000
+HOST = 'localhost'
+WEB_TEST_FILE_SIZE = 1024
+WEB_FILE_TEMP_DIR = tempfile.TemporaryDirectory(dir=os.getcwd())
```

Comment:
use 
```with tempfile.TemporaryDirectory() as tmpdirname``` pattern, it will guarantee files cleanup

Reply:
fyi, yes, it now follows this tempfile usage pattern in the new test case TestIterableDataPipeHttp